transformers/uws/server.js

```diff
@@ -46,7 +46,12 @@ module.exports = function server() {
       });
     });
 
-    this.service.upgrade(soc._handle.fd, req.headers['sec-websocket-key'], soc.ssl ? soc.ssl._external : null);
+    var ticket = this.service.transfer(soc._handle.fd, soc.ssl ? soc.ssl._external : null);
+    soc.on('close', function(error) {
+      if (!error) {
```

Comment:
Yes, so given that `socket.destroy()` is called on the same tick how can an error occur?